src/data/_contributors.yaml

```diff
@@ -265,17 +265,6 @@ samchen:
   description:
     en: Sam is a Front End Developer
 
-hongchanchoi:
```

Comment:
Why did you delete honchangchoi? If this is an unexpected change, please revert.

Reply:
Wow, sorry, I don't know how it happen! 
I did commit with revert,  look at it, please.